internal/engine/local_target_build_and_deployer.go

```diff
@@ -61,9 +61,12 @@ func (bd *LocalTargetBuildAndDeployer) run(ctx context.Context, c model.Cmd) err
 	cmd := exec.CommandContext(ctx, c.Argv[0], c.Argv[1:]...)
 	cmd.Stdout = writer
 	cmd.Stderr = writer
+	if wd != "" {
```

Comment:
I don't think this does anything. If `Dir` is empty the command runs relative to the working directory, which is what the default behavior is anyways, since the empty value for a string in a struct is `""`.